web/src/recent_view_ui.ts

```diff
@@ -1172,16 +1166,17 @@ function recenter_focus_if_off_screen(): void {
 
     if (topic_offset !== "visible") {
         // Get the element at the center of the table.
-        const position = table_wrapper_element.getBoundingClientRect();
-        const topic_center_x = (position.left + position.right) / 2;
-        const topic_center_y = (position.top + position.bottom) / 2;
+        const thead_props = $("#recent-view-table-headers")[0]!.getBoundingClientRect();
```

Comment:
Probably better to not call `getBoundingClientRect` a second time -- we already did it above.